packages/dev/core/src/Cameras/camera.ts

```diff
@@ -922,36 +949,78 @@ export class Camera extends Node {
                 reverseDepth
             );
         } else {
+            const computeObliqueDistance = (camera: Camera) => {

```

Comment:
I think you should not pass the camera and use `this` everywhere.

Mixing `camera` and `this` is not easy to read/understand.

Reply:
We are treating this like a helper function. Using `this` will result in having to write `this as Camera as TargetCamera` since TypeScript will complain about `this as TargetCamera`.

Reply:
Indeed, so why not doing it the other way around, replacing `this` with `camera`, so that the helper only uses the `camera` parameter and doesn't mix with `this`?

Reply:
Oh, I didn't realize it was using `this`. Yeah, that should be `camera`.